karma.conf.js

```diff
@@ -33,6 +33,9 @@ module.exports = function(config) {
 
     webpack: {
       module: {
+       preLoaders: [
+         { test: /\.jsx?$/, loader: "eslint", exclude: /(node_modules)/ }
```

Comment:
Note to self: explain why we want to use preloaders as we might as well put the ESLint loader in the loaders section below.